#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_qt_include.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
##
## DP: Make Qt include path usable in Debian Woody and Sid

@DPATCH@
Index: roadmap/src/qt/Makefile
===================================================================
RCS file: /cvsroot/roadmap/roadmap/src/qt/Makefile,v
retrieving revision 1.10
diff -u -3 -p -u -r1.10 Makefile
--- roadmap/src/qt/Makefile	13 Jun 2004 18:25:47 -0000	1.10
+++ roadmap/src/qt/Makefile	14 Nov 2004 09:29:19 -0000
@@ -1,5 +1,5 @@
 
-GUICFLAGS=-I/usr/include/qt3
+GUICFLAGS=-I/usr/include/qt -I/usr/include/qt3
 GUILDFLAGS=-lqt
 MOC=/usr/bin/moc
 INSTALLDIR=/usr/local
